samples/CallWithChat/src/app/views/CallScreen.tsx

```diff
@@ -78,7 +79,12 @@ export const CallScreen = (props: CallScreenProps): JSX.Element => {
         tooltipText: 'Custom Background'
       }
     ];
-    return { videoBackgroundImages: videoBackgroundImages };
+    return {
+      videoBackgroundOptions: {
+        videoBackgroundImages,
+        onResolveDependency: onResolveVideoEffectDependencyLazy
```

Comment:
I remember Dom proposed some names in the discussion, please check chat history about this